packages/core/test/tooltip/tooltipTests.tsx

```diff
@@ -159,7 +158,7 @@ describe("<Tooltip>", () => {
     function renderTooltip(props?: Partial<TooltipProps>) {
         return mount<TooltipProps>(
             <Tooltip content={<p>Text</p>} hoverOpenDelay={0} {...props} usePortal={false}>
-                <Button id={TEST_TARGET_ID} text="target" />
```

Comment:
this isn't needed - just leave it

Reply:
it doesn't do anything, is never used

Reply:
it's used here - we have no lint rule against defining constants separately and using once so there's no need to make this change

Reply:
I'm saying it's never used by any test

Reply:
But sure, reverted